### ISSUE TITLE
lib.dom.d.ts incorrectly types the return type of HTMLMediaElements.play method

### ISSUE THREAD

**Issue by majames** [microsoft/TypeScript#15691]
**TypeScript Version:**  2.3.2

**Code**

```ts
    /**
      * Pauses the current playback and sets paused to TRUE. This can be used to test whether the media is playing or paused. You can also use the pause or play events to tell whether the media is playing or not.
      */
    pause(): void;
    /**
      * Loads and starts playback of a media resource.
      */
    play(): void;
```

**Expected behavior:**

lib.dom.d.ts should type the returned value of `play()` as `Promise<void>`: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play

**Actual behavior:**

lib.dom.d.ts types the return type as `void`

**Comment by majames** [comment]
Link to PR: https://github.com/Microsoft/TypeScript/pull/15699

**Comment by mhegazy** [comment]
PRs welcomed. You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by SaschaNaz** [comment]
Note that W3C and WHATWG spec disagrees here. It is defined as `void` on W3C spec and as `Promise<void>` on WHATWG spec.

**Comment by majames** [comment]
Thanks for the quick feedback! I've created the new PR here: https://github.com/Microsoft/TSJS-lib-generator/pull/249

**Comment by saschanaz** [comment]
This can now be closed, right?

**Comment by jakebailey** [comment]
Fixed in #16005.
